man/systemd.network.xml

```diff
@@ -1829,6 +1829,14 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>DHCPv6ClientWithoutFlags=</varname></term>
+          <listitem>
+            <para>Takes a boolean. When true, the DHCPv6 client will be started even if there is no
+           managed or other information flag in the RA. Defaults to false.</para>
```

Comment:
Please instead fold that into the DHCPv6Client= option. Add DHCPv6Client=always with this meaning.